app/components/tooltip-presets.ts

```diff
@@ -8,5 +10,47 @@ export default {
       icon: DwollaBalanceIcon,
       title: "Dwolla Balance",
     },
+    "dwolla-connect": {
+      content:
+        "Dwolla Connect leverages your existing banking relationships and corresponding commercial bank accounts to integrate modern payment processing capabilities into your operations.",
+      icon: DwollaConnectIcon,
+      title: "Dwolla Connect",
+    },
+    "external-party": {
+      content:
+        "A type of Dwolla account, either business or personal, that identifies your end-users. An External Party is required in order to initiate a transfer to a customer’s bank account.",
+      title: "External Party",
+    },
+    "funding-source": {
+      content:
+        "A linked bank account that can be used to route funds to a Dwolla Account or External Party, denoted by the use of an account and routing number, most often connected to a checking or savings account.",
+      title: "Funding Source",
+    },
+    "idempotency-key": {
+      content:
+        "Use a unique key in the API request header to prevent duplicate operations in Dwolla.",
+      title: "Idempotency key",
+    },
+    "openapi-spec": {
+      content:
+        "A standardized interface description for HTTP/REST APIs, facilitating understanding without source code access. Can be combined with additional tooling, such as generating documentation, a Postman collection, or a Software Development Kit.",
+      title: "Open API Specification",
+    },
+    sandbox: {
+      content:
+        "A simulation of Dwolla's production environment, complete with all API endpoints, but without real banking or user data.",
```

Comment:
Suggested edit:
A simulated environment where you can imitate production/“live” use cases, safely test your API requests and responses in a controlled setting, and mock out low-volume and high volume data.